app/controllers/v0/intent_to_files_controller.rb

```diff
@@ -37,7 +37,12 @@ def index
         feature_toggle: ApiProviderFactory::FEATURE_TOGGLE_INTENT_TO_FILE
       )
       type = params['itf_type'] || 'compensation'
-      response = intent_to_file_provider.get_intent_to_file(type, nil, nil)
+      if Flipper.enabled?(:disability_compensation_production_tester, @current_user)
+        Rails.logger.info("ITF GET call skipped for user #{@current_user.uuid}")
```

Comment:
For future enhancement we discussed possibly adding a common string for all log messages related to this flipper, making it easier to search thru the logs.